src/util/iroha-helpers/query-helper.js

```diff
@@ -0,0 +1,103 @@
+import { sign as signQuery, derivePublicKey } from 'ed25519.js'
+import cloneDeep from 'lodash/cloneDeep'
+import each from 'lodash/each'
+import { Signature } from '../proto/primitive_pb'
+import * as Queries from '../proto/queries_pb'
+
+const emptyQuery = () => new Queries.Query()
+
+/**
+ * Returns payload from the query or a new one
+ * @param {Object} query
+ */
+const getOrCreatePayload = query => query.hasPayload() ? cloneDeep(query.getPayload()) : new Queries.Query.Payload()
+
+/**
+ * Capitalizes string
+ * @param {String} string
+ * @returns {String} capitalized string
+ */
+const capitalize = string => string.charAt(0).toUpperCase() + string.slice(1)
+
+/**
+ * Returns new query with added command.
+ * @param {Object} query base query
+ * @param {String} queryName name of a query. For reference, visit http://iroha.readthedocs.io/en/latest/api/queries.html
+ * @param {Object} params query parameters. For reference, visit http://iroha.readthedocs.io/en/latest/api/queries.html
+ */
+const addQuery = (query, queryName, params) => {
+  let payloadQuery = new Queries[capitalize(queryName)]()
+
+  each(params, (value, key) => {
```

Comment:
I found that lodash `each` is slower that native `forEach` [lodash.each vs ES6 FOR vs Native](https://www.measurethat.net/Benchmarks/Show/3529/0/lodasheach-vs-es6-for-vs-native---v2). If performance is required for this we can change it with another option, also if we will change it, then we can remove import from lodash.

Also due to #21 we have introduced FP-style partially, so need to use `lodash/fp` for that reason. [Lodash FP](https://github.com/lodash/lodash/wiki/FP-Guide)

Reply:
Speed is not an issue in that case. I've imported lodash/fp correctly now :)